pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -65,7 +65,9 @@ type GCECloud struct {
 	service           *compute.Service
 	containerService  *container.Service
 	projectID         string
-	zone              string
+	region            string
+	instanceZone      string
+	managedZones      []string
```

Comment:
I believe you're taking over this PR, do you want me to add these comments?

In case it isn't clear...
instanceZone = the zone which this instance is in
managedZones = all zones which kubernetes is managing (i.e. all zones in the region for Ubernetes-Lite, just the current zone if we're not running Ubernetes-Lite)